packages/project-access/test/command/npm-command.test.ts

```diff
@@ -0,0 +1,182 @@
+            value: originalPlatform
+        });
+    });
+
+    test('Install node module, no options', async () => {
+        // Mock setup
+        const processMock = getProcessMock((event, cb) => {
+            if (event === 'exit') {
+                cb();
+            }
+        });
+        const spawnMock = jest.spyOn(childProcessMock, 'spawn').mockReturnValueOnce(processMock);
+
+        // Test execution
+        const stdout = await execNpmCommand(['install', '@scope/module@1.2.3']);
+
+        // Result check
+        expect(stdout).toBe('data-STDOUT_MOCK_DATA');
+        expect(spawnMock).toHaveBeenCalledWith('npm', ['install', '@scope/module@1.2.3'], {});
+    });
+
+    test('Install node module, cwd and logger option ', async () => {
+        // Mock setup
+        const processMock = getProcessMock((event, cb) => {
+            if (event === 'exit') {
+                cb();
+            }
+        });
+        const spawnMock = jest.spyOn(childProcessMock, 'spawn').mockReturnValueOnce(processMock);
+        const logger = {
+            info: jest.fn(),
+            error: jest.fn()
+        } as unknown as Logger;
+
+        // Test execution
+        await execNpmCommand(['install', 'mock-module'], { cwd: 'some/path', logger });
+
+        // Result check
+        expect(spawnMock).toHaveBeenCalledWith('npm', ['install', 'mock-module'], { cwd: 'some/path' });
+        expect(logger.error).toBeCalledWith(expect.stringContaining('data-STDERR_MOCK_DATA'));
+        expect(logger.info).toBeCalledWith(expect.stringContaining('data-STDOUT_MOCK_DATA'));
+    });
+
+    test('Error handling with logger', async () => {
+        // Mock setup
+        const processMock = getProcessMock((event, cb) => {
+            if (event === 'error') {
+                cb(Error('ERROR_MOCK'));
+            }
+        });
+        jest.spyOn(childProcessMock, 'spawn').mockReturnValueOnce(processMock);
+        const logger = {
+            error: jest.fn()
+        } as unknown as Logger;
+
+        // Test execution
+        try {
+            await execNpmCommand(['mock-command'], { logger });
+            expect('Function execNpmCommand should have thrown exception but did not').toBe('Error');
+        } catch (error) {
+            expect(error.message).toContain('ERROR_MOCK');
+        }
+        expect(logger.error).toBeCalledWith(expect.stringContaining('ERROR_MOCK'));
+    });
+});
+
+describe('Test execNpmCommand(), simulate windows', () => {
+    beforeAll(() => {
+        Object.defineProperty(process, 'platform', {
+            value: 'win'
+        });
+    });
+
+    beforeEach(() => {
+        jest.resetAllMocks();
+    });
+
+    afterAll(() => {
+        Object.defineProperty(process, 'platform', {
+            value: originalPlatform
+        });
+    });
+
+    test('Install node module with logger', async () => {
+        // Mock setup
+        Object.defineProperty(process, 'platform', {
+            value: 'darwin'
```

Comment:
Shouldn't this be `win` for windows tests, or be removed as defined in `beforeAll`?
```suggestion
            value: 'win'
```

Reply:
good catch, I've switched later to a complete block with mocking `process.platform`, was probably a leftover. Fixed with https://github.com/SAP/open-ux-tools/pull/2066/commits/a8122363522a7e855668cc42c5e4cb3ccedda2c0